translate/src/modules/entities/actions.ts

```diff
@@ -67,6 +68,25 @@ export const getEntities =
     dispatch({ type: REQUEST_ENTITIES });
 
     const content = await fetchEntities(location, page);
+    const {
+      search,
+      search_exclude_source_strings,
+      search_identifiers,
+      search_match_case,
+      search_match_whole_word,
+      search_rejected_translations,
+    } = location;
+
+    if (search) {
+      logUXAction('Search Options Selected', 'Search Option Statistics', {
```

Comment:
These would be silghtly more consistent with existing action and experiment names:

```suggestion
      logUXAction('Load: String list with search parameter', 'Search Options Statistics', {
```